cdap-data-fabric/src/main/java/io/cdap/cdap/store/StoreDefinition.java

```diff
@@ -120,8 +134,13 @@ public static final class NamespaceStore {
             .build();
 
     public static void create(StructuredTableAdmin tableAdmin) throws IOException {
-      createIfNotExists(tableAdmin, NAMESPACE_TABLE_SPEC);
-      createIfNotExists(tableAdmin, REPOSITORY_TABLE_SPEC);
+      register();
+      createIfNotExists(tableAdmin);
```

Comment:
Some junits around 10-15 call these create() method individually, just to setup the individual stores which are needed for that particular test.

Hence we need to retain the create() method.

Reply:
Probably a nit, but if this is just for tests can we keep the existing method as is with the intent to cleanup tests later.

New code can use `register()` and `createRegisteredTables()`(just rename the existing method you have) methods.

I'll leave this up to you.

Reply:
Keeping this as is for now, because if we keep the createIfNotExists method as well as register(), both of them will have code duplicates where we specify the tables related to that particular store. 

But suggestion to renaming to createRegisteredTables() sounds good! Have updated accordingly.